docs/errors.rst

```diff
@@ -3,12 +3,14 @@
 Error Source
 ============
 
-For the most part, error messages will come directly from the database with the same `structure that PostgreSQL uses <https://www.postgresql.org/docs/current/plpgsql-errors-and-messages.html>`_, PostgREST will convert the ``MESSAGE``, ``DETAIL``, ``HINT`` and ``ERRCODE`` from the PostgreSQL error to JSON format and add an HTTP status code to the response (see :ref:`status_codes`). For instance, this is the error you will get when querying a nonexistent table:
+For the most part, error messages will come directly from the database with the same `structure that PostgreSQL uses <https://www.postgresql.org/docs/current/error-style-guide.html>`_. PostgREST will convert the ``MESSAGE``, ``DETAIL``, ``HINT`` and ``ERRCODE`` from the PostgreSQL error to JSON format and add an HTTP status code to the response (see :ref:`status_codes`). For instance, this is the error you will get when querying a nonexistent table:
 
 .. code-block:: http
 
   GET /nonexistent_table?id=eq.1 HTTP/1.1
 
+  404 Not Found
+
 .. code-block:: json
 
   {
```

Comment:
```suggestion
.. code-block:: http

  HTTP/1.1 404 Not Found
  Content-Type: application/json; charset=utf-8
```

Maybe doing this would be more accurate (separate the request from the response, though perhaps more cluttered).